tracer/src/Datadog.Trace/Ci/Configuration/CIVisibilitySettings.cs

```diff
@@ -11,15 +12,60 @@ internal class CIVisibilitySettings
     {
         public CIVisibilitySettings(IConfigurationSource source)
         {
-            Enabled = source?.GetBool(ConfigurationKeys.CIVisibilityEnabled) ?? false;
-            if (Enabled)
-            {
-                TracerSettings = new TracerSettings(source);
-            }
+            Enabled = source?.GetBool(ConfigurationKeys.CIVisibility.Enabled) ?? false;
+            Agentless = source?.GetBool(ConfigurationKeys.CIVisibility.AgentlessEnabled) ?? false;
+            ApiKey = source?.GetString(ConfigurationKeys.ApiKey);
```

Comment:
Given this is already available on `TracerSettings` and `TelemetrySettings`, and now here, maybe we should move it onto `TracerSettings`? 

Same goes for `Site`, as it's already available on those?

(Could do in a separate PR if it makes sense obviously)

Reply:
Let's do this in another PR.